resources/cluster-essentials/files/functions.serverless.crd.yaml

```diff
@@ -3,7 +3,6 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.2.4
-    "helm.sh/resource-policy": keep
```

Comment:
@magicmatatjahu is this annotation still needed?

Reply:
I would keep it for the next release.